transaction/core/src/constants.rs

```diff
@@ -22,8 +22,8 @@ pub const MAX_TOMBSTONE_BLOCKS: u64 = 100;
 /// The MobileCoin network will contain a fixed supply of 250 million mobilecoins (MOB).
 pub const TOTAL_MOB: u64 = 250_000_000;
 
-/// Minimum allowed fee, denominated in picoMOB.
-pub const BASE_FEE: u64 = 10;
+/// Minimum allowed fee, denominated in milliMOB (Note, base unit is picoMOB).
```

Comment:
Wouldn't the denomination of `MINIMUM_FEE` still be picoMOB? The `10` is denominated in `milliMOB` but the doc comment is meant to document the `MINIMUM_FEE` variable itself.

If someone was looking at autogenerated docs they might get the impression that `MINIMUM_FEE` itself is denominated in milliMOB and thus requires multiplying by `1_000_000_000` in order to be denominated in picoMOB.

Reply:
Ah thank you - I originally was going to do the conversion elsewhere. Will fix up the comment.